src/cc_helpers.cc

```diff
@@ -505,60 +505,80 @@ void certifier::framework::cc_trust_data::clear_sensitive_data() {
   // Not necessary on most platforms.
 }
 
+//  cc_trust_data relies on the following data in the store
+//    public_policy_key
+//    public_key_algorithm
+//    symmetric_key_algorithm
+//    for attestation:
+//      service-attest-key
+//      sealing-key
+//    for authentication
+//      auth-key
+//      app-symmetric-key
+//    Some platforms require obtaining a platform rule. For them, it should
```

Comment:
In the phrase, "In them, it ..." ... the word 'it' is ambiguous. How about rephrasing this as:

"platform-rule (signed-claim) is also stored for platforms that require a platform-rule."

Is that clearer?